### PR TITLE
Update modal designs

### PR DESCRIPTION
**Pull request by antonyshchenko**
Screenshot 2024-11-18 at 15 58 25
Screenshot 2024-11-18 at 16 02 28

### CODE REVIEW

**Review by bholzman — APPROVED** [review]
Looks good!

**Comment by bholzman** [comment]
Actually, in the design the modal veil applies a blur to the background, didn't see that in here

**Comment by antonyshchenko** [comment]
Added blur:

Screenshot 2024-11-18 at 16 32 43

File: web-ui/tailwind.config.mjs [hunk 1]
```diff
@@ -41,9 +41,10 @@ export default {
       }
     },
     colors: {
-      darkestBackground: "#141414",
-      darkerBackground: "#161616",
-      darkBackground: "#1B1B1B",
+      darkBackground4: "#141414",
+      darkBackground3: "#151515",
+      darkBackground2: "#161616",
+      darkBackground1: "#1B1B1B",
```
Comment (antonyshchenko) [inline web-ui/tailwind.config.mjs:47]: @bholzman note that I've changed these

**Pull request merged**